text/0124-documents-api.md

```diff
@@ -75,7 +80,7 @@ Sets the maximum number of documents to be returned by the current request.
 
 ###### 3.1.1.2.3. `fields`
 
-- Type: String
+- Type: Array of Strings
```

Comment:
Not for the `GET` endpoint AFAIK. That's why separating the description of `GET indexes/:index_uid/documents` and `POST indexes/:index_uid/documents/fetch` can be helpful to handle every aspect of each endpoint.

Reply:
Huum, yes, it is; it's not the same syntax, but the get route also accepts an array of strings.
I'm not sure I see the point of duplicating ~150 lines of specification if both routes are pretty much the same.
I'm really in favor of following what we did with the two search routes we have.